src/main/java/bisq/core/dao/node/parser/TxOutputParser.java

```diff
@@ -81,8 +82,9 @@ public void processGenesisTxOutput(TempTx genesisTx) {
         }
     }
 
-    void processOpReturnCandidate(TempTxOutput txOutput) {
+    boolean processOpReturnCandidate(TempTxOutput txOutput) {
```

Comment:
If we return a boolean we should rename to make that more clear (e.g. isOpReturnCandidate)